components/greetings/greetings.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./greetings.module.css";
+
+const date = new Date();
+const currentTime = date.getHours();
+
+let greetings;
+
+if (currentTime < 12) {
+  greetings = "Good Morning";
+} else if (currentTime < 18) {
+  greetings = "Good Afternoon";
+} else if (currentTime < 22) {
+  greetings = "Good Evening";
+} else {
+  greetings = "Good Night";
+}
```

Comment:
Good job!
I would create a function from this (maybe `createGreetings()`?) and call it in the `Greetings` component.

Reply:
Thank you! I will open a Pull Request for it